course/include/isometry.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+
+#include <initializer_list>
+#include <string>
+#include <sstream>
+#include <cmath>
+#include <iostream>
+
+namespace ekumen {
+  namespace math {
+
+class Vector3 {
+ public:
+  static const Vector3 kUnitX;
+  static const Vector3 kUnitY;
+  static const Vector3 kUnitZ;
+  static const Vector3 kZero;
+  Vector3(double x, double y, double z): x_(x), y_(y), z_(z) {};
+  Vector3():x_(0.),y_(0.),z_(0.){};
+  double x() const {return x_};
+  double y() const {return y_};
+  double z() const {return z_};
+  double& x() {return x_;};
+  double& y() {return y_;};
+  double& z() {return z_;};
+  double norm() const;
+  double & operator [] (const int index) const;
+  double & operator [] (const int index);
+  Vector3 operator + (const Vector3 & obj) const;
```

Comment:
Also define the `+=`, `-=`, etc operators. Later you should use those to implement `+`, `-`, etc.